From 17bde33ffe50be558520a063392dc73b319a7f20 Mon Sep 17 00:00:00 2001
From: Yangbo Lu <yangbo.lu@nxp.com>
Date: Thu, 28 Nov 2019 11:48:24 +0800
Subject: [PATCH] LF-368 net: mscc: ocelot: hard code VCAP_PORT_CNT for Felix
 temporarily

The Felix switch supports different port number with Ocelot.
This makes the bits mapping in TCAM entry and Action entry
is different with Ocelot for VCAP function.

Signed-off-by: Yangbo Lu <yangbo.lu@nxp.com>
---
 drivers/net/ethernet/mscc/ocelot_vcap.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/net/ethernet/mscc/ocelot_vcap.h
+++ b/drivers/net/ethernet/mscc/ocelot_vcap.h
@@ -25,7 +25,7 @@
 #define VCAP_IS2_CNT 64
 #define VCAP_IS2_ENTRY_WIDTH 376
 #define VCAP_IS2_ACTION_WIDTH 99
-#define VCAP_PORT_CNT 11
+#define VCAP_PORT_CNT 6
 
 /* IS2 half key types */
 #define IS2_TYPE_ETYPE 0
